react-app/src/library/services/vendor-implementations/plantronics/plantronics.ts

```diff
@@ -104,6 +141,7 @@ export default class PlantronicsService extends VendorImplementation {
           if (response.status === 404) {
             if (isRetry) {
               plantronicsInstance.isConnected = false;
+              plantronicsInstance.$lastStatusUpdate.next('isConnected')
```

Comment:
Why are you emitting `isConnected` if we receive a `404`?

Reply:
+1